kitchen-ec2.gemspec

```diff
@@ -23,7 +23,8 @@ Gem::Specification.new do |gem|
   gem.add_dependency "test-kitchen", "~> 1.4", ">= 1.4.1"
   gem.add_dependency "excon"
   gem.add_dependency "multi_json"
-  gem.add_dependency "aws-sdk", "~> 2"
+  gem.add_dependency "aws-sdk-iam"
```

Comment:
prefer to depend on a major version like `~> 1.2` so we don't bump across major versions would testing them